modules/oe_whitelabel_helper/oe_whitelabel_helper.module

```diff
@@ -92,7 +92,13 @@ function oe_whitelabel_helper_requirements_alter(array &$requirements): void {
   if (isset($requirements['slim_select_library'])) {
     $theme_handler = \Drupal::service('theme_handler');
     $theme_path = $theme_handler->getTheme('oe_bootstrap_theme')->getPath();
-    $path = '/' . $theme_path . '/assets/js/slimselect.min.js';
+    $version = \Drupal::config('slim_select.settings')->get('version');
```

Comment:
Let's move all sim select code into a class under /source SlimSelectProcess (or if u have a better idea). Mark it as internal. Also this part that you added for version 2 repeats, so you can add it into a private method.

Reply:
ok please check my changes.